astropy/modeling/functional_models.py

```diff
@@ -2014,3 +2014,118 @@ def evaluate(cls, x, y, amplitude, r_eff, n, x_0, y_0, ellip, theta):
         z = np.sqrt((x_maj / a) ** 2 + (x_min / b) ** 2)
 
         return amplitude * np.exp(-bn * (z ** (1 / n) - 1))
+
+class CartesianShaplets(Fittable2DModel):
+    r"""
+    Two dimensional cartesian shapelets model.
+    Also computes Fourier Transform of shapelets.
+
+    Parameters
+    ----------
+    beta0 : float
+        Characteristic scale parameter for first
+    dimension basis functions.
```

Comment:
Indent all the lines in the description to match the first one otherwise sphinx assumes ``dimension`` is a parameter, too.